common/src/main/kotlin/org/sandboxpowered/silica/vanilla/network/PacketBase.kt

```diff
@@ -1,6 +1,7 @@
 package org.sandboxpowered.silica.vanilla.network
 
 interface PacketBase {
-    fun read(buf: PacketByteBuf)
+    @Deprecated("use PacketByteBuf constructor instead", ReplaceWith("Unit"))
```

Comment:
I wouldn't use a ReplaceWith here as that would produce odd code

Reply:
I'm not sure why it wants that there, but it complains without it.